docs/collecting-data/code-generation/using-the-cli/index.md

```diff
@@ -16,13 +16,18 @@ The Snowtype CLI is a tool which aims to speed up tracking implementations, prov
 
 A Console API key is required for the Snowtype CLI to authenticate with your account. You can find your own or create one in the Console [API key management](https://console.snowplowanalytics.com/credentials).
 
-From there, the ways for the CLI to recognize the key are either through the global `-k, --apiKey` option or the `SNOWPLOW_CONSOLE_API_KEY` environment variable. The Snowtype CLI does automatically read from a `.env` file at the root of your project as well.
+:::note
+Both API key and API key ID variables are required for versions > `0.9.0`.
+:::
+
+From there, the ways for the CLI to recognize the key are either through the global `-k, --apiKey` and `-s, apiKeyId` options or the `SNOWPLOW_CONSOLE_API_KEY` and `SNOWPLOW_CONSOLE_API_KEY_ID` environment variables. The Snowtype CLI does automatically read from a `.env` file at the root of your project as well.
```

Comment:
maybe you can rephrase it a bit something like (removing From there):

```
The CLI can recognize the key either through the global -k, --apiKey and -s, --apiKeyId options, or via the SNOWPLOW_CONSOLE_API_KEY and SNOWPLOW_CONSOLE_API_KEY_ID environment variables. Additionally, the Snowtype CLI automatically reads from a .env file located at the root of your project.
```

Reply:
Adjusted, thanks!